### PR TITLE
AODN_NSW_OEH multi-beam pipeline update

### PR DESCRIPTION
**Pull request by mhidas**
* Add unit tests
* Update required shapefile attributes
* Add more shapefile checks

### CODE REVIEW

**Comment by mhidas** [comment]
@pblain Could this be post-reviewed in the next iteration please?

**Comment by pblain** [comment]
Sure!

**Comment by mhidas** [comment]
@pmbohm Here's a test zip file that should get through this pipeline without problems: 
[NSWOEH_20151029_PortHackingBateBay_MB.zip](https://github.com/aodn/data-services/files/1107832/NSWOEH_20151029_PortHackingBateBay_MB.zip)
In the PO box, copy it into `/var/incoming/NSW-OEH/incoming/`

**Comment by mhidas** [comment]
Thanks @pmbohm Yeah Macs do annoying things with their hidden files. I'll add a check to make sure there's only one shapefile, but in your example it would be caught anyway because the extra one doesn't match the required file name convention.

File: AODN/NSW-OEH/process_zip.py [hunk 1]
```diff
@@ -65,12 +63,14 @@ def check_crs(crs_field):
...
     """
     Return a tuple consisting of
-    1) a list of uncerscore-separated fields in the file name, and
+    1) a list of underscore-separated fields in the file name, and
     2) the file name extension (part of name after the first '.')
 
     """
     file_name = os.path.basename(path)
-    name, extension = file_name.split('.', 1)
+    split = file_name.split('.', 1)
```
Comment (mhidas) [inline AODN/NSW-OEH/process_zip.py]: Sure. I'll add this to my next PR.

**Comment by pmbohm** [comment]
@mhidas Ive run out of time for this but its looking good. I renamed a file insode  a test zip to `tar.gz` and that was ok as well.
Maybe another post review could be done with the new version your working on?

**Comment by pmbohm** [comment]
With this variable:
`all_extensions = ('zip', 'xyz', 'xya', 'tif', 'tiff', 'sd', 'kmz', 'pdf', 'tar.gz') + shapefile_extensions`
And this file in the zip 
`NSWOEH_20151029_PortHackingBateBay_MB_BTYGRD010GSS_W84Z56AHD_FLD744_20151221_FV02.tar`
All tests passed

**Pull request merged**